src/clpp11.hpp

```diff
@@ -355,6 +356,12 @@ class Device {
            std::string{"."} + std::to_string(GetInfo<cl_uint>(CL_DEVICE_COMPUTE_CAPABILITY_MINOR_NV));
   }
 
+  // Returns if the Nvidia chip is a Volta or later archicture (sm_70 or higher)
+  bool IsPostNVIDIAVolta() const {
```

Comment:
CLBlast also has a CUDA back-end, which works because every function in this file is also implemented in `cupp11.h`, so you'll have to mimic this behaviour with the same API in the other file as well to make the CUDA version still work.

Reply:
Ah, I didn't notice! I can do that.